cmd/bridge-auth-config-update.go

```diff
@@ -112,6 +112,6 @@ func init() {
 	cmd := updateAuthorizerConfigCmd
 	rootCmd.AddCommand(cmd)
 
-	cmd.PersistentFlags().String("id", "", "authorizer ID")
+	cmd.PersistentFlags().String("id string", "", "authorizer ID")
```

Comment:
do you think it really works?  I don't see any instruction to add `type` in variable name on https://github.com/spf13/cobra.

Reply:
It wouldn't work, btw, we don't need to do it for other types except `bool`.